core/01_core.js

```diff
@@ -105,18 +110,24 @@
   } = ensureFastOps();
 
   // core/infra collaborative code
-  delete window.Deno.core.__setOpCallTracingEnabled;
-  delete window.Deno.core.__initializeCoreMethods;
-  delete window.Deno.core.__resolvePromise;
+  delete window.__infra;
+
   __initializeCoreMethods(
     eventLoopTick,
-    submitOpCallTrace,
+    submitLeakTrace,
   );
 
-  function submitOpCallTrace(id) {
+  function submitLeakTrace(id) {
     const error = new Error();
-    ErrorCaptureStackTrace(error, submitOpCallTrace);
-    op_opcall_tracing_submit(id, StringPrototypeSlice(error.stack, 6));
+    ErrorCaptureStackTrace(error, submitLeakTrace);
+    op_leak_tracing_submit(0, id, StringPrototypeSlice(error.stack, 6));
```

Comment:
These magic numbers for kind could use comments

Reply:
Ah yeah, this was copied from the old opcall code. It's `"Error\n".length`. I can add a comment.